test-requirements.txt

```diff
@@ -0,0 +1 @@
+freezegun
```

Comment:
Why this?

Reply:
This is necessary to avoid the runboat action error indicating that dependency is missing.